commons/src/main/java/io/hyscale/commons/commands/provider/ImageCommandProvider.java

```diff
@@ -44,29 +49,15 @@
 	private static final String REMOVE_IMAGE = "rmi";
 	private static final String PULL_COMMAND = "pull";
 	private static final String LABEL_ARGS = "label";
-	private static final String IMAGE_OWNER = "imageowner";
-	private static final String HYSCALE = "hyscale";
 	private static final String HYPHEN = "-";
 	private static final String FORCE_FLAG = "f";
 	private static final String QUIET = "q";
-	private static final String FILTER = "filter";
+	private static final String FILTER = "--filter";
 	private static final boolean USE_SUDO = false;
 	private static final String TARGET = "target";
-
-    public String getBuildImageName(String appName, String serviceName) {
-        StringBuilder sb = new StringBuilder();
-        sb.append(HYSCALE_IO_URL).append(SLASH).append(appName).append(SLASH).append(serviceName);
-        return normalize(sb.toString());
-    }
-
-    public String getBuildImageNameWithTag(String appName, String serviceName, String tag) {
-        StringBuilder sb = new StringBuilder();
-        sb.append(HYSCALE_IO_URL).append(SLASH).append(appName).append(SLASH).append(serviceName);
-        if (StringUtils.isNotBlank(tag)) {
-            sb.append(ToolConstants.COLON).append(tag);
-        }
-        return normalize(sb.toString());
-    }
+	
+	@Autowired
+	private ImageMetadataUtil imageMetadataUtil;
```

Comment:
This name request has not been resolved. 
Provide a better name as referred to in the previous comments.

Reply:
As per the previous comment, the name should reflect the scope of the class which is restricted to the Image metadata, hence the name ImageMetadataUtil. Any better suggestions?

Reply:
I would suggest "ImageMetadataProvider".
I don't see this class as a utility. What do you say?